src/routes/community/+page.svelte

```diff
@@ -114,7 +114,7 @@
     ];
 
     const metrics = [
-        { metric: '39K+', description: 'GitHub Stars' },
+        { metric: '40K+', description: 'GitHub Stars' },
```

Comment:
we have this as a constant now to avoid having to update it in so many places:

```suggestion
        { metric: `${GITHUB_STARS}+`, description: 'GitHub Stars' },
```